tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1221,6 +1221,88 @@ some text
 
 `;
 
+exports[`inline2.php 1`] = `
+<?php
+// test
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+// test
+?>
+<h1>Test</h1>
+
+`;
+
+exports[`inline3.php 1`] = `
+<h1>Test</h1>
+<?php
+// test
+?>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<h1>Test</h1>
+<?php
+// test
+?>
+`;
+
+exports[`inline4.php 1`] = `
+<?php
+/**
+ * A
+ */
+
+// B
+
+/**
+ * C
+ */
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+/**
+ * A
+ */
+// B
+/**
+ * C
+ */
+?>
+<h1>Test</h1>
+
+`;
+
+exports[`inline5.php 1`] = `
+<h1>Test</h1>
+<?php
+// test
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<h1>Test</h1>
+<?php  ?><?php
```

Comment:
This comes from the `printLines` function - any idea how to handle this case best @evilebottnawi?

Reply:
Don't know need search solution